### ISSUE TITLE
Changelog for runtime 0.12.0

### ISSUE THREAD

**Issue by greggb** [facebook/regenerator#351]
Is there a changelog for the minor bump from a few days ago? 

I only see 4 releases with notes after #272. Did they move?

**Comment by benjamn** [comment]
You're not missing anything. I tend to document my changes in the descriptions of the relevant PRs, which are all linked in the recent commit history (and which link to the relevant issues, in turn). Paraphrasing those changes in a separate changelog has never felt useful to me, though I understand others may find it valuable in larger projects with more commit activity.

Just out of curiosity, what kind of information are you looking for? Have you encountered a bug that you're trying to bisect/diagnose? I'm more than happy to answer specific questions here!

**Comment by greggb** [comment]
Makes sense. Mostly it's when I run `yarn upgrade-interactive` and see that there is a minor or major update, I want to make sure there won't be a breaking change.

Sometimes it's nice to know if there is an improvement I can take advantage of in the release.

For this one specifically I noticed it was a minor release not a patch and wanted to see the reason for that. I checked through the release commits, but they were just updating the version strings. Nothing specific I was looking for, just like to gather info on my deps.

Thanks for the quick reply

**Comment by benjamn** [comment]
Here's the PR that affected `regenerator-rutnime`: https://github.com/facebook/regenerator/pull/350

If you're using async generator functions (with both `await` and `yield`), `yield`ing a `Promise` will now be equivalent to `yield await <promise>`, which is a spec behavior that changed since my original implementation. I think most people use a separate transform to handle async generators (before Regenerator runs), so these changes may not matter much, even if you're using async generators.
